### PR TITLE
Modify package name to yelp-gprof2dot, update description and version

### PR DESCRIPTION
**Pull request by mjbryant**
I'd like to get this package on our internal PyPI so I can use it easily in a service. To do that I need to change the package name. Tested this locally by installing the package via `pip install -e /path/to/working/dir` and finding the proper script in place.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,15 +15,16 @@
 from setuptools import setup
 
 setup(
-    name='gprof2dot',
-    version='2015.02.03',
+    name='yelp-gprof2dot',
+    version='1.0.0',
     author='Jose Fonseca',
```
Comment (ArnaudBrousseau) [inline setup.py]: Modify this to "Yelp Inc." if we're planning to maintain this going forward?

File: setup.py [hunk 2]
```diff
@@ -15,15 +15,16 @@
...
     author_email='jose.r.fonseca@gmail.com',
```
Comment (ArnaudBrousseau) [inline setup.py]: I'd get rid of this, doesn't make a lot of sense

File: setup.py (hunk 1 above)
Comment (mjbryant) [inline setup.py]: Sure. I didn't see this pattern repeated on some of our other forks I looked at, like [hacheck](https://github.com/Yelp/sqlparse/blob/master/setup.py) and [kafka-python](https://github.com/Yelp/kafka-python/blob/master/setup.py), so I left it the way it is. As long as I leave the copyright alone it should be fine, though.

File: setup.py (hunk 2 above)
Comment (mjbryant) [inline setup.py]: Get rid of what? You appear to have commented on lines 15-21.

File: setup.py [hunk 3]
```diff
@@ -10,20 +10,23 @@
...
-    name='gprof2dot',
-    version='2015.02.03',
-    author='Jose Fonseca',
-    author_email='jose.r.fonseca@gmail.com',
-    url='https://github.com/jrfonseca/gprof2dot',
-    description="Generate a dot graph from the output of several profilers.",
+    name='yelp-gprof2dot',
+    version='1.0.0',
+    author='Yelp Performance Team',
+    author_email='team-perf@yelp.com',
```
Comment (ArnaudBrousseau) [inline setup.py]: I meant get rid of the "author_email" key entirely. I don't think team-perf is reachable from outside is it?

Comment (mjbryant) [inline setup.py]: Oh, right. Done.

**Pull request merged**